README.md

```diff
@@ -8,24 +8,90 @@ ospool:
+	$ condor_q -hold
+	ap40.uw.osg-htc.org : <128.105.68.62:9618?... @ 08/01/24 15:01:50
+ 	ID         OWNER          HELD_SINCE  HOLD_REASON
+	130.0       alice           8/1  14:56 Transfer output files failure at ⋯
 
-	$ condor_q -better-analyze JOB-ID -pool POOL-NAME
+In this particular case, the message indicates that HTCondor cannot transfer back output files, likely because they were never created. This hold condition can be triggered if a user has this option in their HTCondor submit file:
+
+	transfer_output_files = outputfile
+
+However, when the job executed, it went into the Held state. To learn more, `condor_q -hold JOB-ID` can be used. An example from this scenario is shown below:
+
+	$ condor_q -hold JOB-ID
+
+	Transfer output files failure at access point… while receiving files from the execution point. Details: Error from ….execute point … failed to send file(s) at apxx; failed to read from file /path: (errno 2) No such file or directory
+
+The reason why the file transfer failed is because `outputfile` was never created on the worker node. Remember that at the beginning we said that the user specifically requested `transfer_outputfiles = outputfile`! HTCondor could not complete this request, and so the job went into the Held state instead of finishing normally.
+
+It's quite possible that the error was simply transient, and if we retry, the job will succeed. We can re-queue a job that is in Held state by using `condor_release`: 
+
+	condor_release JOB-ID 
+
+### Common scenarios for job going on hold
+
+Some of the common scenarios and example hold mesaages where the job goes on hold are described below
+<ul>
+<li> The executable is using more resources(memory) than requested in the job submit file. An example `hold` message for this scenario will look like:</li>
+	
+	Error from …: memory usage exceeded request_memory
+	Job in status 2 put on hold by SYSTEM_PERIODIC_HOLD due to memory usage ...
+
+<li> Typo(s) in the `transfer_input_files` path: If there are typos in the file path or the file does not exist in the provided path, then HTCondor will put the job on hold and will provide a hold message similar to</li>
+
+	Transfer input files failure at access point apxx while sending files to the execution point. Details: reading from file /path: (errno 2) No such file or directory
+
+<li> If a job ran more than the default allowed run time-HTCondor will put the job on hold providing a message like</li>
+
+	The job exceeded allowed execute duration of 20:00:00
+
+<li> You have reached your disk quota and HTCondor can not write the output file to your access point. This may happen while performing [checkpointing](https://portal.osg-htc.org/documentation/htc_workloads/submitting_workloads/checkpointing-on-OSPool/) jobs or regular jobs when your disk quota is reached. The hold message will look like:</li>
```

Comment:
html and markdown formatting can't mix. Because this list item is "wrapped" in HTML, the links will also need to use html syntax and not markdown.